index.html

```diff
@@ -63,9 +72,9 @@ <h1>My Recent Works</h1>
             <div class="indicator-image">
             <img src="./img/Indicator.svg" alt="Indicator">
             </div>
-
-            <div class="work-container">
-                <article>
+            <!-- <div id="modal-container"></div> -->
+            <div class="work-container" id="work-container-id">
```

Comment:
It's important to follow the exact Figma template, so please make sure to fix your design on the mobile screens.

- [ ] 1. The actual button is getting out of its original place, and the `X` icon on the image can't be used for closing.
- [ ] 2. The picture on the desktop screen is quite large, you can limit the height and width using `max-width` and `max-height` properties.

![image](https://user-images.githubusercontent.com/34779928/195699671-b8bc1675-31ab-4c93-981a-3987aee4b82a.png)